input/data/2005/2005A%26A...439.1013A/tev-000061.yaml

```diff
@@ -1,4 +1,25 @@
 source_id: 61
 reference_id: 2005A&A...439.1013A
 
-# TODO: add data
+data:
+  livetime: 58.6
+  excess: 2469
+  significance: 21
+
+pos:
+  ra: {val: 13h3m0.4s, err: 4.4s, err_sys: 20s}
+  dec: {val: -63d11m55s, err: 31s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.16, err: 0.02}
+  sigma2: {val: 0.16, err: 0.02}
```

Comment:
Remove `sigma2` and `TODO` comment for `pa`.
(this was a symmetric gaussian fit and there's just one extension `sigma`.